sources/packages/backend/libs/sims-db/src/entities/student-assessment.model.ts

```diff
@@ -260,6 +260,24 @@ export class StudentAssessment extends RecordDataModel {
     referencedColumnName: ColumnNames.ID,
   })
   relatedApplicationAssessment?: StudentAssessment;
+  /**
+   * Last reported assessment id corresponding to the offering date change.
+   */
+  @ManyToOne(() => StudentAssessment, { nullable: true })
```

Comment:
I notice that eager and cascade are not set to false(we are setting it explicitly in most  instances in our code) and from the documentation I do not see any side effect of not setting it as false. 

@andrewsignori-aot @guru-aot @andrepestana-aot @lewischen-aot FYI.